share/mdnjs/parse.py

```diff
@@ -214,6 +216,18 @@ def writerows(self):
           'disambiguation': disambig
         })
       for mdn in self.inverted_index[keyword]:
+        # add redirect for Web/Api pages
+        if any( word in mdn.title for word in self.CLASS_WORDS ) and len(mdn.title.split('.')) > 1:
```

Comment:
@jdorweiler Sorry for another nitpick which I haven't noticed before. 

I think that doing a split on `mdn.title` and then looking at the number of parts of the split could be changed for `'.' in mdn.title`.

Or did I miss something?